scripts/deploy.ts

```diff
@@ -57,15 +57,19 @@ async function main() {
     case 42161:
     // tarbeth
     case 421614:
-      walletImplementationContractName = 'ArbethWalletSimple';
-      contractPath = `contracts/coins/${walletImplementationContractName}.sol:${walletImplementationContractName}`;
+      walletImplementationContractName = 'WalletSimple';
```

Comment:
Any specific reason for deploying WalletSimple instead of ArbethWalletSimple?

Reply:
aren't both the same ?

Reply:
Ohh makes sense, yes